api-reference/beta/api/application_list_extensionproperties.md

```diff
@@ -1,8 +1,8 @@
 # List extensionProperties
 
-Retrieve a list of extensionproperty objects.
+Retrieve a list of extensionProperty objects.
```

Comment:
MS Graph Extensions and Schema Extensions are the future. This extensionProperty should become legacy. Since there are considerable usage of this in existing apps getting rid of this will block the migration story. We have got be careful updating this. So here is what I suggest.
1. Block the usage further by not listing creation scenario (api-reference/beta/api/application_post_extensionproperties.md)
2. Let the this list scenario be there for now (i.e. api-reference/beta/api/application_list_extensionproperties.md) and update the description that this would return the Azure AD Directory Schema Extension properties.